<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OxygenConfigurationUI</class>
 <widget class="QWidget" name="OxygenConfigurationUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>548</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Tit&amp;le alignment:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>titleAlignment</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="titleAlignment">
        <item>
         <property name="text">
          <string>Left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Center</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Center (Full Width)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Right</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>199</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>B&amp;utton size:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>buttonSize</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="drawBorderOnMaximizedWindows">
        <property name="text">
         <string>Display window borders for maximized windows</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="buttonSize">
        <item>
         <property name="text">
          <string comment="@item:inlistbox Button size:">Small</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Button size:">Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Button size:">Large</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="@item:inlistbox Button size:">Very Large</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QCheckBox" name="drawSizeGrip">
        <property name="text">
         <string>Add handle to resize windows with no border</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Animations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="animationsDurationLabel">
        <property name="text">
         <string>&amp;Animations duration:</string>
        </property>
        <property name="buddy">
         <cstring>animationsDuration</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="animationsDuration">
        <property name="suffix">
         <string>ms</string>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="animationsEnabled">
        <property name="text">
         <string>Enable animations</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Window-Specific Overrides</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="Oxygen::ExceptionListWidget" name="exceptions" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Oxygen::ExceptionListWidget</class>
   <extends>QWidget</extends>
   <header>config/oxygenexceptionlistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
